rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java

```diff
@@ -45,7 +46,12 @@ public Optional<Coin> getLockingCap() {
     }
 
     @Override
-    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) {
+    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) throws BridgeIllegalArgumentException {
+        if (newLockingCap.getValue() <= 0) {
+            logger.warn("[increaseLockingCap] {} {}", "Locking Cap must be greater than zero. Value attempted: ", newLockingCap.value);
+            throw new BridgeIllegalArgumentException("Locking Cap must be greater than zero");
```

Comment:
```suggestion
            String message = String.format("Locking cap must be greater than zero. Value attempted: %d", newLockingCap.value);
            logger.warn("[increaseLockingCap] {}", message);
            throw new BridgeIllegalArgumentException(message);
```
Avoids repetition

Reply:
Done! Created `INCREASE_LOCKING_CAP_LOG_TAG` constant to avoid `String` repetition. The warning was an issue in the whole method, not only related to this piece of code.

Reply:
i think marcos meant the `"Locking cap must be greater than zero. Value attempted: %d"` part

Reply:
@julia-zack It doesn't solve the whole warning. For this reason, I applied a fix that avoid a chain of warnings.

Reply:
but you still have that line duplicated

Reply:
Created a `message` String variable to avoid repetition in: `"Locking cap must be greater than zero. Value attempted:"`